project/Dependencies.scala

```diff
@@ -7,7 +7,8 @@ import sbt._
 object Dependencies {
   val scala212 = sys.props.getOrElse("scala212.nightly", "2.12.18")
   val scala213 = sys.props.getOrElse("scala213.nightly", "2.13.12")
-  val scala3 = sys.props.getOrElse("scala3.nightly", "3.4.0-RC2")
+  val scala3LTS = sys.props.getOrElse("scala3.nightly", "3.3.2-RC4")
```

Comment:
`RemoveUnused` tests red until 3.3.3 which will export unused warnings to semanticdb